common/pgmetrics/templates/deployment.yaml

```
@@ -40,7 +40,7 @@ spec:
         imagePullPolicy: IfNotPresent
         env:
         - name: DATA_SOURCE_NAME
-          value: postgresql://postgres:{{ $db_pass }}@{{ $db_host }}:5432/{{ $db_name }}?sslmode=disable
+          value: postgresql://postgres:{{ $db_pass }}@{{ $db_host }}:5432/{{ $db_name }}?sslmode=disable&application_name=pgmetrics
```

I suggest to use the same name as the deployment for clarity:
```suggestion
          value: postgresql://postgres:{{ $db_pass }}@{{ $db_host }}:5432/{{ $db_name }}?sslmode=disable&application_name={{ .Release.Name }}-pgmetrics
```